### DISCUSSION TITLE
Is it possible to use same viz_type for different plugins?

### DISCUSSION THREAD

**Discussion by kiddten** [apache/superset#10718] [General]
I am building my own plugin with some customization for standard pivot table plugin. So the I want to be able use and standard pivot table chart and mine customized too.
The problem is that for my plugin I need data as that requested from backend in standard plugin with `viz_type = pivot_table`.
As I understood `new PivotTableChartPlugin().configure({ key: 'pivot_table' }),` that key responsible for viz_type  mapping.
So If I am doing something like this `new PivotTableMineChartPlugin().configure({ key: 'pivot_table_mine' })`, I am getting error from backend, since there is no viz_type for my custom key.

In case `new PivotTableMineChartPlugin().configure({ key: 'pivot_table' })` standard plugin replaced with my custom.
To summarize I need use both plugins with same viz_type..

**Comment by rusackas** [comment]
these keys do indeed need to be unique. Closing since this is years old, but I hope you solved it :)
